packages/lib/attributeClass/cache.ts

```diff
@@ -17,8 +18,11 @@ export const attributeClassCache = {
     byEnvironmentIdAndName(environmentId: string, name: string) {
       return `environments-${environmentId}-name-${name}-attributeClasses`;
     },
+    byCountAndEnvironmentId(environmentId: string) {
```

Comment:
I think for simplicity I would choose to use the `byEnvironmentId` also for the count because otherwise it's one more tag to revalidate under certain conditions. Disadvantage is that we also invalidate the cache when an attribute class is updated (the count isn't changed), but since this is very uncommon ( I think also not supported at the moment?) let's just go with `byEnvironmentId` for both list and count.